mne_bids/write.py

```diff
@@ -331,10 +335,14 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     """
     # get measurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (tuple, list, np.ndarray)):
+    if meas_date is None or anonymize is not None:
+        acq_time = 'n/a'
```

Comment:
if `anonymize is not None`, then the user wants to anonymize a date that is potentially there. With this code you would set the acq_time to 'n/a' ... even if meas_date is NOT None.

anonymization doesn't mean to set the measurement date to "n/a", but to shift it back by some amount of time so that the true time becomes unknown.

Reply:
Agreed! When anonymize is not `None` a `daysback` parameter is passed which tells you where to shift the meas date to. It's not possible to completely erase the date from fif files.